src/util/graph-util.js

```diff
@@ -56,7 +56,7 @@ function statementToNT (statement, excludeDot) {
 function graphFromStatements (statements, rdf) {
   var graph = rdf.graph()
   statements.forEach(function (st) {
-    graph.addStatement(st)
+    graph.add(st)
```

Comment:
This was bizarre. The `addStatement` method was not updating the indices and breaking querying. Will report that bug.